packages/terra-framework-docs/src/terra-dev-site/doc/table/Examples/ZebraStripedTable.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Table from 'terra-table';
+
+const gridDataJSON = {
```

Comment:
Non-blocking nit: This is table but here and other places using this data this is referencing grid.